cookbooks/bcpc-hadoop/recipes/hbase_repl.rb

```diff
@@ -15,5 +16,27 @@
       echo "remove_peer '#{node["bcpc"]["hadoop"]["hbase"]["repl"]["peer_id"]}'"|hbase shell
       EOH
     only_if "echo list_peers|hbase shell|grep #{node["bcpc"]["hadoop"]["hbase"]["repl"]["peer_id"]}"
+    user 'hbase'
+  end
+end
+
+#
+# To disable region replication peer
+#
+if (node['bcpc']['hadoop']['hbase']['site_xml']['hbase.region.replica.replication.enabled'] == 'false')
+  bash "disable_hbase_region_repl_peer" do
+    code <<-EOH
+      echo "disable_peer 'region_replica_replication'"|hbase shell
+      EOH
+    only_if "echo list_peers|hbase shell|grep region_replica_replication|grep ENABLED"
+    user 'hbase'
+  end
+else
+  bash "enable_hbase_region_repl_peer" do
+    code <<-EOH
+      echo "enable_peer 'region_replica_replication'"|hbase shell
+      EOH
+    only_if "echo list_peers|hbase shell|grep region_replica_replication|grep DISABLED"
```

Comment:
Either way you invoke hbase shell to make a test, is there any harm to just blindly disabling without testing?